docs/_templates/index.html

```diff
@@ -0,0 +1,508 @@
+                  </li>
+                  <li class="col-6-12 ae-4">
+                    <h3 class="margin-top-3">Strong</h3>
+                    <p class="tiny opacity-6">State-of-the-art, better than most.</p>
+                  </li>
+                  <li class="col-6-12 ae-5">
+                    <h3 class="margin-top-3">Ease of use</h3>
+                    <p class="tiny opacity-6">Get the models with one line of code.</p>
+                  </li>
+                </ul>
+              </li>
+              <li class="col-1-12">&nbsp;</li>
+            </ul>
+          </div>
+
+        </div>
+      </div>
+    </div>
+    <div class="background" style="background-image:url(_static/assets/img/background/img-01.jpg)"></div>
+  </section>
+
+
+
+  <!-- Slide 5 (#85) -->
+  <section class="slide fade-6 kenBurns">
+    <div class="content">
+      <div class="container">
+        <div class="wrap">
+
+          <div class="fix-7-12">
+            <h1 class="ae-1">Win-Win Solution</h1>
+            <p class="opacity-8">GluonCV embrace flexible development pattern while is super easy to optimize and deploy without retaining heavy weight deep learning framework.</p>
+            <ul class="grid grid-85 equal margin-top-5 ae-2 fromCenter" data-equal-collapse-width="767">
+              <li class="col-6-12 col-tablet-1-2 col-phablet-1-1 margin-bottom-2 ae-3">
+                <div class="pad" style="background: #F5FBFE">
+                  <h3>Flexbile</h3>
+                  <!-- <div class="price ae-5"><span class="currency">$</span>0</div> -->
+                  <!-- <div class="ae-5">
+                    <h6 class="uppercase bold small opacity-4">No Credit Card Needed</h6>
+                  </div> -->
+                  <div class="margin-top-3 margin-bottom-3 equalElement ae-6">
+                    <ul class="p tiny">
+                      <li><strong>Develop in Python</strong></li>
+                      <li class="opacity-8">Customizable</li>
+                      <li class="opacity-8">Easy to debug</li>
+                    </ul>
+                  </div>
+                  <!-- <a href="#" class="button green gradient wide cropSides cropBottom ae-7">Try Free</a> -->
+                </div>
+              </li>
+              <li class="col-6-12 col-tablet-1-2 col-phablet-1-1 ae-4">
+                <div class="pad">
+                  <h3>Deploy Friendly</h3>
+                  <!-- <div class="price ae-6"><span class="currency">$</span>6</div> -->
+                  <!-- <div class="ae-6">
+                    <h6 class="uppercase bold small opacity-4">Billed per Month</h6>
+                  </div> -->
+                  <div class="margin-top-3 margin-bottom-3 equalElement ae-7">
+                    <ul class="p tiny">
+                      <li><strong>Export to non-python</strong></li>
+                      <li><strong>Mobile-Optimized</strong></li>
+                      <li><strong>No heavy dependency</strong></li>
```

Comment:
Can we add something like **High Performance** since gluoncv has been integrated with so many accelerate libraries like Intel DNNL, Horovod, NCCL, DALI, etc.

Reply:
good point